impl/sql/mutationstorage/queue.go

```diff
@@ -29,29 +30,74 @@ import (
 	pb "github.com/google/keytransparency/core/api/v1/keytransparency_go_proto"
 )
 
+// AddShards creates and adds new shards for queue writing to a domain.
+func (m *Mutations) AddShards(ctx context.Context, domainID string, shardIDs ...int64) error {
+	glog.Infof("mutationstorage: AddShard(%v, %v)", domainID, shardIDs)
+	for _, shardID := range shardIDs {
+		if _, err := m.db.ExecContext(ctx,
+			`INSERT INTO Shards (DomainID, ShardID, Enabled)  Values(?, ?, ?);`,
+			domainID, shardID, true); err != nil {
+			return err
```

Comment:
Great point. This could change to `INSERT IGNORE`. Unfortunately  `INSERT IGNORE` doesn't have the same syntax in SQLite as it does in MySQL...  

I'll add a TODO for this and to change the testing database off of SQLite

Reply:
SGTM.